docs/general/user-defined-functions.rst

```diff
@@ -59,17 +60,18 @@ These functions can be created like so::
     +---------------+
     SELECT 1 row in set (... sec)
 
-Arguments can be named in the function definition.
-
-For example, if you wanted two ``geo_point`` arguments named ``start_point``
-and ``end_point``, you would do it like this::
-
-    cr> CREATE OR REPLACE FUNCTION calculate_distance(start_point geo_point, end_point geo_point)
-    ...  RETURNS real
-    ...  LANGUAGE JAVASCRIPT
-    ...  AS 'function calculate_distance(start_point, end_point){
-    ...        return Math.sqrt( Math.pow(end_point[0] - start_point[0], 2), Math.pow(end_point[1] - start_point[1], 2));
-    ...      }';
+It is possible to use named function arguments in the function signature. For
+example, the ``calculate_distance`` function signature has two arguments of
+the ``geo_point`` data type that are named as ``start`` and ``end``::
```

Comment:
this could be simplified with a little rewording. for instance:

"It is possible to use named function arguments in the function signature. For example, the `calculate_distance` function signature has two `geo_point` arguments named `start` and `end`:"